app/assets/stylesheets/to_change_everything/_quebecois.scss

```diff
@@ -0,0 +1,64 @@
+.quebecois {
+    @import "quebecois/font_overrides";
+    /*--- Highlight language in mobile language menu ---*/
+    /*--- TODO: make sure the div selection (e.g. #lang1) matches what was added in
+          app/views/layouts/to_change_everything.html.erb ---*/
+    #lang1  li  > a[href="/tce/quebecois"] {
+        color: #d62d34 !important;
+    }
+
+    .check-quebecois:before {
+        content: '';
+    }
+
+    #leaders, #answering, #liberation, #control, #relationships, #profit {
```

Comment:
the CSS in this file is mostly just trying to get all of the text in the paragraphs to not overflow into the next section for any of the 2-column layouts